test/commands/backlog/.fixtures/checkin.test.ts.-backlogcheckin-branch=true,pr=false.json

```diff
@@ -93,7 +93,7 @@
     {
         "scope": "https://bwa.nrs.gov.bc.ca:443",
         "method": "GET",
-        "path": "/int/jira/rest/api/2/search?fields=fixVersions,issuetype,project,status&jql=fixVersion++%3D+15604+AND+issuetype+%3D+RFC+AND+statusCategory+!%3D+Done",
```

Comment:
Fixtures files are auto generated, manually updating them is problematic as it will be keep coming back. Can you remove this from now, and we look into investigate why they are being generated this way?